dat/create_reference_tables.py

```diff
@@ -0,0 +1,82 @@
+# script is run with the pyspark-322-delta-200 environment
+# create the env with this command: conda env create -f envs/pyspark-322-delta-200
+# activate the environment with this command: conda activate pyspark-322-delta-200
+
+import pyspark
+from delta import *
+
+builder = (
+    pyspark.sql.SparkSession.builder.appName("MyApp")
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+    .config(
+        "spark.sql.catalog.spark_catalog",
+        "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+    )
+)
+
+spark = configure_spark_with_delta_pip(builder).getOrCreate()
+
+
+def create_reference_table1():
+    columns = ["letter", "number", "a_float"]
+    data = [("a", 1, 1.1), ("b", 2, 2.2), ("c", 3, 3.3)]
+    rdd = spark.sparkContext.parallelize(data)
+    df = rdd.toDF(columns)
+
+    df.write.format("delta").save("reference_tables/reference_table1/table")
```

Comment:
If the reference table can be generated by the script, then they should be .gitignored no?

Reply:
@edmondo1984 - Think we'd like this repo to be easily clone-able, so users can simply clone the repo to access the reference tables without running the generation scripts themselves.  Thoughts?

Reply:
Having reference tables actually exist is preferable; I'd rather not have to regenerate the tables in CI jobs that are running DAT tests.